notebooks/analysis/notebooks/Actions.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "240e130f-4cae-4955-a8ed-a692f5ea9092",
+   "metadata": {},
+   "source": [
+    "## Figuring out what users do when they are searching is made easier by identifying actions"
```

Comment:
The aim of the notebook is a little unclear to me, and I imagine it would be for new readers too! Can you give a bit more context in the introduction?

Reply:
I've rejigged this accordingly. Sending another commit.